index.html

```diff
@@ -40,7 +47,7 @@ <h1>I'm Naan. <br>Glad to see <br>you!</h1>
 
     <!--Works Section-->
     <div class="works">
-        <div class="card_works">
+        <div class="card_works work_1">
             <div class="snapshot_portfolio"></div>
             <div class="left_block underline">
                 <div class="project_title">Tonic</div>
```

Comment:
- [OPTIONAL] It is required to stick to the Figma design as much as possible. This includes font, font-weight, color, background-color, margin, padding among others. Your project works do not fall to the center of the page. Kindly look into that and fix it.

### Your Design
![your-single-proj-spacing](https://user-images.githubusercontent.com/26597432/221377791-73f8047b-9fbd-4f2a-bb00-349662df1892.png)

### Figma Design 
![temp-single-proj-spacing](https://user-images.githubusercontent.com/26597432/221377801-77a9510c-9c96-4f59-aa53-89a8ab8c589e.png)